### ISSUE TITLE
Accessibility: navigate submenus with keyboard

### ISSUE THREAD

**Issue by joehobson** [navnorth/wp-oet-theme#2]
When a user is accessing this site with a keyboard only, they cannot navigate the main menu’s submenus.
